framework/packages/abstract-client/src/client.rs

```diff
@@ -102,6 +105,34 @@ impl<Chain: CwEnv> AbstractClient<Chain> {
             .map_err(Into::into)
             .map_err(Into::into)
     }
+
+    // Retrieve the last account created by the client.
+    /// Returns `None` if no account has been created yet.
+    pub fn get_last_account(&self) -> AbstractClientResult<Option<Account<Chain>>> {
```

Comment:
Woud be nice to mention that it would only load accounts stored in this system

Reply:
done!